x/slashing/spec/02_state.md

```diff
@@ -44,31 +52,23 @@ The information stored for tracking validator liveness is as follows:
 // ValidatorSigningInfo defines a validator's signing info for monitoring their
 // liveness activity.
 message ValidatorSigningInfo {
+  option (gogoproto.equal)            = true;
+  option (gogoproto.goproto_stringer) = false;
+
   string address = 1;
-  // height at which validator was first a candidate OR was unjailed
+  // Height at which validator was first a candidate OR was unjailed
   int64 start_height = 2;
-  // index offset into signed block bit array
+  // Index which is incremented each time the validator was a bonded
+  // in a block and may have signed a precommit or not. This in conjunction with the
+  // `SignedBlocksWindow` param determines the index in the `MissedBlocksBitArray`.
   int64 index_offset = 3;
-  // timestamp validator cannot be unjailed until
+  // Timestamp until which the validator is jailed due to liveness downtime.
   google.protobuf.Timestamp jailed_until = 4;
-  // whether or not a validator has been tombstoned (killed out of validator
-  // set)
+  // Whether or not a validator has been tombstoned (killed out of validator set). It is set
+  // once the validator commits an equivocation or for any other configured misbehiavor.
   bool tombstoned = 5;
-  // missed blocks counter (to avoid scanning the array every time)
-  int64 missed_blocks_counter = 6;
+  // A counter kept to avoid unnecessary array reads.
+  // Note that `Sum(MissedBlocksBitArray)` always equals `MissedBlocksCounter`.
+  int64 missed_blocks_counter = 6 [(gogoproto.moretags);
 }
 ```
-
-Where:
```

Comment:
Isn't it better to have these fields explained in a separate section like the old one?

Reply:
They are exlained directly in the structure comments (source code).

Reply:
Yeah I was just wondering if, considering that this is developer-facing docs, it could be more appropriate and readable to have it in a separate section - as it was before. Not a big deal anyways

Reply:
Yes - having it in a bulletpoint list makes sense, but here it's clearly a repetition.